### PR TITLE
LIT 1811 6156

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Revisions. The entire story is quite complex and requires a deeper study; I now arranged the records provisionally, according to my current understanding. (There are a few things in the article of Knibb that remain unclear to me).

### CODE REVIEW

**Comment by eu-genia** [comment]
Judging by the abstract
```
<abstract>
              <p>The work shares with
                <ref type="work" corresp="LIT1811Liveso"/> the chronological part at the beginning, which
                is however not included into the edition <bibl><ptr target="bm:Knibb1985LivesProphets2"/>.
                </bibl>

              </p>
          </abstract>
```
you were actually planning to edit 6156, not 1811?

**Comment by DenisNosnitsin1970** [comment]
Well, what is "planning" in this case... Does it change anything?

**Comment by eu-genia** [comment]
I mean it sounds strange that the work 1811 says that it shares something with work 1811

File: 1001-2000/LIT1811Liveso.xml [hunk 1]
```diff
@@ -53,28 +53,62 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
         <div type="bibliography">
             <listRelation>
-                <relation name="lawd:hasAttestation" active="LIT6156OriginProphets" passive="BNFabb195"/>
+                <relation name="lawd:hasAttestation" active="LIT1811Liveso" passive="BNFabb195"/>
+                <relation name="saws:isRelatedTo" active="LIT1811Liveso" passive="LIT6156OriginProphets"/>
             </listRelation>
                </div>
             <div type="edition">
               <note>The division into text parts follows <ref type="mss" corresp="BNFabb195"/>.</note>
                <div type="textpart" subtype="chapter" xml:id="Chronology">
```
Comment (thea-m) [inline 1001-2000/LIT1811Liveso.xml:98]: xml:lang="gez" should be added here (also below)

**Review by thea-m — CHANGES_REQUESTED** [review]

**Comment by DenisNosnitsin1970** [comment]
Corrections, additions done.

**Review by thea-m — APPROVED** [review]

**Pull request merged**